src/index.native.scroll.js

```diff
@@ -35,6 +35,14 @@ const styles = StyleSheet.create({
 class SwipeableViews extends Component {
   static propTypes = {
     /**
+     * If true, changes to the index prop will cause an
+     * animated transition, otherwise it'll just jump to
+     * the new slide.
+     *
+     * TODO: Implement this on iOS
+     */
+    animateTransitions: React.PropTypes.bool,
```

Comment:
So in my case: I have infinite scroll implemented with slide 0 representing the last slide in my deck. When i get to it I want to jump to the end without animation